libbeat/_meta/config.reference.yml

```diff
@@ -227,6 +227,11 @@ output.elasticsearch:
   # IPv6 addresses should always be defined as: https://[2001:db8::1]:9200
   hosts: ["localhost:9200"]
 
+  # Enabled ilm to use index lifecycle management instead daily indices.
+  #ilm.enabled: false
+  #ilm.rollover_alias: "beat-index-prefix"
+  #ilm.pattern: "00001"
```

Comment:
Why is the `pattern` configurable, how do you expect users to change that?

Reply:
Haven't tested it yet but users could also use date pattern for it: https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-rollover-index.html#_using_date_math_with_the_rollover_api